### PR TITLE
refactor LOG and add comment for meta, meta/processors/job

### PR DESCRIPTION
**Pull request by liwenhui-soul**
#### What type of PR is this?
- [ ] bug
- [ ] feature
- [ ] enhancement

#### What problem(s) does this PR solve?
Issue(s) number: 

Description:

#### Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

File: src/meta/processors/job/JobExecutor.h [hunk 1]
```diff
@@ -22,32 +22,74 @@ class JobExecutor {
   explicit JobExecutor(kvstore::KVStore* kv) : kvstore_(kv) {}
   virtual ~JobExecutor() = default;
 
-  // Check the arguments about the job.
+  /**
+   * @brief Check the arguments about the job.
```
Comment (pengweisong) [inline src/meta/processors/job/JobExecutor.h:26]: If the comment start with capital letter, then all comments should keep consistent? ps. And I think make first letter capital is better.

File: src/meta/ActiveHostsMan.h [hunk 2]
```diff
@@ -107,30 +117,101 @@ class ActiveHostsMan final {
   ~ActiveHostsMan() = default;
 
   using AllLeaders = std::unordered_map<GraphSpaceID, std::vector<cpp2::LeaderInfo>>;
+
+  /**
+   * @brief save host info and leader info into kvStore
```
[the remark this answers is not in the document]
Comment (critical27) [reply src/meta/ActiveHostsMan.h:122]: 👍

File: src/meta/MetaServiceUtils.h [hunk 3]
```diff
@@ -20,29 +20,73 @@ class MetaServiceUtils final {
  public:
   MetaServiceUtils() = delete;
 
+  /**
+   * @brief change column definition in memory
+   *
+   * @param cols existed column in schema
+   * @param prop schema property, mainly used to check if the col is ttl_col
+   * @param col
```
Comment (critical27) [inline src/meta/MetaServiceUtils.h:28]: This method is really hard to understand, I have to read the code every time 🤣 
Maybe add some description, about this function, e.g. `col` is the new column definition, `op` is the ADD/DROP/ALTER.

**Review by critical27 — COMMENTED** [review]
Maybe make the comments start with Capital letter, some comments in the file is all lower letter. Or at least unify.

**Review by pengweisong — APPROVED** [review]

File: src/meta/MetaServiceUtils.h (hunk 3 above)
Comment (liwenhui-soul) [reply src/meta/MetaServiceUtils.h]: done

File: src/meta/ActiveHostsMan.h [hunk 4]
```diff
@@ -107,30 +117,101 @@ class ActiveHostsMan final {
...
+   * time
+   *
+   * @param kv from where to get the host's information
+   * @param host which host to check
+   * @return
+   */
   static ErrorOr<nebula::cpp2::ErrorCode, bool> isLived(kvstore::KVStore* kv, const HostAddr& host);
 
+  /**
+   * @brief get hostInfo for a host
```
[the remark this answers is not in the document]
Comment (liwenhui-soul) [reply src/meta/ActiveHostsMan.h]: > Could add some explanations: Will test if host has been registered or not.

it doesn't check this in fact

File: src/meta/ActiveHostsMan.h [hunk 5]
```diff
@@ -107,30 +117,101 @@ class ActiveHostsMan final {
...
+   * @return
+   */
   static ErrorOr<nebula::cpp2::ErrorCode, std::vector<HostAddr>> getActiveHosts(
       kvstore::KVStore* kv, int32_t expiredTTL = 0, cpp2::HostRole role = cpp2::HostRole::STORAGE);
 
+  /**
+   * @brief get services in the agent host
+   *
+   * @param kv from where to get
+   * @param hostname
```
[the remark this answers is not in the document]
Comment (liwenhui-soul) [reply src/meta/ActiveHostsMan.h]: > hostname or ip

done

File: src/meta/ActiveHostsMan.h (hunk 2 above)
[the remark this answers is not in the document]
Comment (liwenhui-soul) [reply src/meta/ActiveHostsMan.h]: have added more comments
but , I think it is a common function and heartbeat relative comments should not occur here, so I didn't add it

File: src/meta/ActiveHostsMan.h [hunk 6]
```diff
@@ -107,30 +117,101 @@ class ActiveHostsMan final {
...
+   * @param leaderParts leader info of all parts
+   * @return
+   */
   static nebula::cpp2::ErrorCode updateHostInfo(kvstore::KVStore* kv,
                                                 const HostAddr& hostAddr,
                                                 const HostInfo& info,
                                                 const AllLeaders* leaderParts = nullptr);
 
+  /**
+   * @brief check if the host is registed
```
[the remark this answers is not in the document]
Comment (liwenhui-soul) [reply src/meta/ActiveHostsMan.h]: fixed

File: src/meta/processors/job/CompactJobExecutor.h [hunk 7]
```diff
@@ -18,6 +18,13 @@ class CompactJobExecutor : public SimpleConcurrentJobExecutor {
                      AdminClient* adminClient,
                      const std::vector<std::string>& params);
 
+  /**
+   * @brief
+   *
+   * @param address the host that task command send to
+   * @param parts parts that the host contains
+   * @return
+   */
```
[the remark this answers is not in the document]
Comment (liwenhui-soul) [reply src/meta/processors/job/CompactJobExecutor.h:30]: done

File: src/meta/processors/job/JobExecutor.h (hunk 1 above)
Comment (liwenhui-soul) [reply src/meta/processors/job/JobExecutor.h:26]: done

**Review by critical27 — APPROVED** [review]

File: src/meta/processors/job/LeaderBalanceJobExecutor.h [hunk 8]
```diff
@@ -36,15 +36,43 @@ class LeaderBalanceJobExecutor : public MetaJobExecutor {
...
+   */
   ErrorOr<nebula::cpp2::ErrorCode, bool> buildLeaderBalancePlan(HostLeaderMap* hostLeaderMap,
                                                                 GraphSpaceID spaceId,
                                                                 int32_t replicaFactor,
                                                                 bool dependentOnZone,
                                                                 LeaderBalancePlan& plan,
                                                                 bool useDeviation = true);
 
+  /**
+   * @brief host will loop for the partition which is not leader, and try to acuire the leader
```
Comment (pengweisong) [inline src/meta/processors/job/LeaderBalanceJobExecutor.h:65]: Fist letter

File: src/meta/processors/job/LeaderBalanceJobExecutor.h [hunk 9]
```diff
@@ -70,6 +98,14 @@ class LeaderBalanceJobExecutor : public MetaJobExecutor {
                                                       HostParts& hostParts,
                                                       int32_t& totalParts);
 
+  /**
+   * @brief compare and get the lost hosts and expand hosts
```
Comment (pengweisong) [inline src/meta/processors/job/LeaderBalanceJobExecutor.h:102]: ditto

File: src/meta/processors/zone/MergeZoneProcessor.h [hunk 10]
```diff
@@ -24,9 +27,13 @@ class MergeZoneProcessor : public BaseProcessor<cpp2::ExecResp> {
  private:
   explicit MergeZoneProcessor(kvstore::KVStore* kvstore) : BaseProcessor<cpp2::ExecResp>(kvstore) {}
 
+  /**
+   * @brief get all parts of a space in each host
```
Comment (pengweisong) [inline src/meta/processors/zone/MergeZoneProcessor.h:31]: ditto

**Review by pengweisong — DISMISSED** [review]
Good Job

**Comment by critical27** [comment]
There are conflicts~

**Review by critical27 — APPROVED** [review]

**Review by liuyu85cn — APPROVED** [review]

**Review by critical27 — APPROVED** [review]

**Pull request merged**